src/content/getting-started/management-cluster/index.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Access to the management cluster
+description: The Giant Swarm Management API is easily accessible by means of kubectl-gs, a kubectl plugin developed by Giant Swarm. This tutorial guides you throughout some of the most important features of the tool.
+weight: 30
+menu:
+  main:
+    parent: getting-started
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-honeybadger
+user_questions:
+  - How can I access the management cluster?
+  - How can I access the Giant Swarm Management API?
+last_review_date: 2023-06-12
+---
+
+Giant Swarm provides customers with their own _management cluster (MC)_. You can have multiple management clusters, for example if different cloud providers are used. The management Kubernetes cluster runs operators that create and manage all the workload clusters (WCs). On workload Kubernetes clusters, you deploy and run actual business applications.
```

Comment:
```suggestion
Giant Swarm provides customers with their own _management cluster (MC)_, a regular Kubernetes cluster with some sugar on top. You can have multiple management clusters, for example, if different cloud providers are used. The management cluster runs operators that create and manage all the workload clusters (WCs). On workload clusters, you deploy and run actual business applications.
```

Reply:
well it sounds weird to me to name it `management clusters` and then add the `Kubernetes` as an adjective. If your intention was to explain those are regular k8s clusters, then extra phrase could help. But it is not a blocker just an opinion ;)

Reply:
Right. I've reworded it once more so "Kubernetes" doesn't appear somewhere in between.